File: program/program/convert-decimal-to-octal/convert_decimal_to_octal.php
<?php
    echo "Enter a decimal number: ";
    $decimal_number = trim(fgets(STDIN)); // Get the decimal number input by the user
    $octal_number = decoct($decimal_number); // Convert the decimal number to octal
        
    echo "The octal equivalent of $decimal_number is: $octal_number";
?>

File: program/program/convert-decimal-to-octal/ConvertDecimalToOctal.cs
using System;
class DecimalToOctal {
   public static void Main() {
      int decVal, quot, i = 1, j;
      int[] octalVal = new int[80];
      Console.WriteLine("Enter a Decimal Number: ");
      decVal = Convert.ToInt32(Console.ReadLine());
      quot = decVal;
      while (quot!= 0) {
         octalVal[i++] = quot % 8;
         quot = quot / 8;
      }
      Console.Write("Equivalent Octal Number: ");
      for (j = i - 1; j > 0; j--)
         Console.Write(octalVal[j]);
         Console.Read();
   }
}
